src/components/core/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,98 @@
+import * as Stitches from '@stitches/react';
+import { styled } from 'Styles/stitches.config';
+import { modifyVariantsForStory } from 'Styles/type-utils';
+import BreadcrumnSeperatorIcon from '@assets/svg/breadcrumb-seperator.svg';
+
+type BreadcrumbProps = {
+    items: string[];
+    dark: boolean;
+    handleOnClick: (item: string) => any;
```

Comment:
Please use the exact type instead of any.
Also, please add `@typescript-eslint/no-explicit-any` to eslint rules